content/src/content/jcr_root/apps/core/wcm/components/breadcrumb/v2/breadcrumb/breadcrumb.html

```diff
@@ -17,7 +17,14 @@
      aria-label="${'Breadcrumb' @ i18n}"
      data-sly-use.breadcrumb="com.adobe.cq.wcm.core.components.models.Breadcrumb"
      data-sly-use.template="core/wcm/components/commons/v1/templates.html"
-     data-sly-test="${breadcrumb.items.size > 0}">
+     data-sly-test="${breadcrumb.items.size > 0}"
+     data-sly-test.lastModified="${properties['cq:lastModified'].getTime.toGMTString || properties['jcr:lastModified'].getTime.toGMTString || properties['jcr:created'].getTime.toGMTString}"
```

Comment:
Please don't use `data-sly-test` as it causes the whole element to be skipped/not rendered if the expression evaluates to falsy. You can use `data-sly-set` instead.